src/cmd/build.js

```diff
@@ -27,31 +27,31 @@ export function safeFileName(name: string): string {
 
 // defaultPackageCreator types and implementation.
 
-export type ExtensionBuildResult = {
+export type ExtensionBuildResult = {|
   extensionPath: string,
-};
+|};
 
-export type PackageCreatorParams = {
+export type PackageCreatorParams = {|
   manifestData?: ExtensionManifest,
   sourceDir: string,
   fileFilter: FileFilter,
   artifactsDir: string,
   showReadyMessage: boolean
-};
+|};
 
-export type LocalizedNameParams = {
+export type LocalizedNameParams = {|
   messageFile: string,
   manifestData: ExtensionManifest,
-}
+|}
 
 // This defines the _locales/messages.json type. See:
 // https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales
-type LocalizedMessageData = {
-  [messageName: string]: {
+type LocalizedMessageData = {|
+  messageName: {|
```

Comment:
oh, sorry! This should be `[messageName: string]`. I was confused for some reason. The message file can have many keys of this type: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales